Gemfile

```diff
@@ -22,10 +22,11 @@ end
 # Windows and JRuby does not include zoneinfo files, so bundle the tzinfo-data gem
 # and associated library.
 platforms :mingw, :x64_mingw, :mswin, :jruby do
-  gem "tzinfo", "~> 1.2"
+  gem "tzinfo", "~> 2.0", ">= 2.0.6"
   gem "tzinfo-data"
 end
 
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
+gem "webrick", "~> 1.7"
```

Comment:
Why was this package added?

Reply:
After updating github pages versions and some other versions of gemfile, without this package we can't serve the page at local.

Reply:
A comment should be in the code. Something like we have in CuraCultura:
https://github.com/curacultura/curacultura-website/blob/main/Gemfile#L27-L28